__tests__/app.test.js

```diff
@@ -26,3 +26,20 @@ describe("GET /api", () => {
       })
   })
 })
+
+describe("get /api/topics", () => {
+  test("200: Responds with array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true)
+        topics.forEach((topic) => {
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)